### PR TITLE
30 index menu cards

### LINKED ISSUE #30: Add menu cards to homepage
Add cards with the different food options to the main homepage.

### PR DESCRIPTION
**Pull request by gc824**
Added cards to the landing page. 
closes #30

### CODE REVIEW

File: contactus.php [hunk 1]
```diff
@@ -13,58 +13,81 @@
...
+>>>>>>> master
 				<li class="nav-item">
 					<a class="nav-link" href="contactus.php">Connect with Us!</a>
 				</li>
 				<li class="nav-item">
 					<a class="nav-link" href="recipes.php">Recipes</a>
 				</li>
+<<<<<<< HEAD
                 <li class="nav-item">
 					<a class="nav-link" href="orderform.php">Order form</a>
```
Comment (rbaia28) [inline contactus.php:33]: can we capitalize the F in form?

**Comment by billythakidd04** [comment]
@gc824 Is this still a draft? Is it ready for review?

**Comment by gc824** [comment]
No longer decided to create order form.

**Comment by billythakidd04** [comment]
Delete the branch

**Pull request closed without merging**